fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -0,0 +1,3 @@
+<i>lichess</i> is built for the love of chess. The chess server has 150 000 individual users daily and growing fast. Several chess variants are supported (Crazyhouse, Chess 960, King Of The Hill, Three-check, Antichess, Atomic chess, Horde, Racing Kings), either in online and offline modes!
```

Comment:
Some grammatical suggestions, and make statistics fuzzier to avoid having to update them.

```suggestion
<i>lichess</i> is built for the love of chess. The server hosts hundreds of thousands of users and games every day. Several chess variants are supported (Crazyhouse, Chess 960, King Of The Hill, Three-check, Antichess, Atomic chess, Horde, Racing Kings). It supports both online and offline modes!
```